docs/design.md

```diff
@@ -898,9 +903,11 @@ A +---------------------+
   +---------------------+
   | TLV area            |
   | +-----------------+ |
-  | | TLV area header | | <- struct image_tlv_info
+  | | TLV area header | | <- struct image_tlv_info (optional)
   | +-----------------+ |
-  | | Dependency      | | <- Dependency entry (struct image_tlv)
+  | | Protected TLVs  | | <- Protected TLVs (struct image_tlv)
+  | +-----------------+ |
+  | | TLV area header | | <- struct image_tlv_info
```

Comment:
might put also the unprotected TLV area header magic here

Reply:
Done.